h2oai/h2oai.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "h2oai",
+    "key": "h2oai",
+    "label": "H2O AI",
+    "description": "AI for the Digital Brain",
+    "info": "https://www.h2o.ai/",
```

Comment:
"info" needs to be a URL to a page in the NDS Confluence:
https://nationaldataservice.atlassian.net/wiki/display/NDSC/NDS+Labs+Services

Create a new service entry as a subpage of "NDS Labs Services". Copy the "Template" example and add some very minimal descriptive information (you don't need to provide everything in the template, but certainly how to run the examples as outlined in the JIRA ticket would be nice).